common/Earthfile

```diff
@@ -0,0 +1,62 @@
+VERSION 0.6
+FROM ruby:2.7.1
+WORKDIR /dependabot-common
+
+CREATE_DEPENDABOT_USER:
+    COMMAND
+
+    ARG uid=1000
+    ARG gid=$uid
+
+    ENV DEPENDABOT_NATIVE_HELPERS_PATH=${DEPENDABOT_NATIVE_HELPERS_PATH:-/opt}
+
+    RUN groupadd -f --gid "$gid" dependabot \
+     && if ! id -u dependabot; then \
+            useradd --uid "$uid" --gid "$gid" -m dependabot; \
+            mkdir -p $DEPENDABOT_NATIVE_HELPERS_PATH; \
+            chown dependabot:dependabot $DEPENDABOT_NATIVE_HELPERS_PATH; \
+        fi
+
+CONFIGURE_GIT_USER:
```

Comment:
This user-defined command is run in test targets for ecosystems with specs that rely on a default git user being configured.